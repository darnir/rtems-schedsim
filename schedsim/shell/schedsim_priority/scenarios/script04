echo "*** TEST 04 ***"
rtems_init
echo Create 0x0a010001 
task_create TA1 1
task_suspend TA1
task_create TA1 2
task_resume TA1
echo "*** END OF TEST 04 ***"
exit
# We will not get here
