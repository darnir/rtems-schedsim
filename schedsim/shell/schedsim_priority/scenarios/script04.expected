Script File               : script04.scen
verbose                   : 1
==> 1: echo "*** TEST 04 ***"
*** TEST 04 ***
==> 2: rtems_init
  CPU 0: Thread Heir: 0x09010001 priority=255
  CPU 0: Thread Executing: 0x09010001 priority=255
==> 3: echo Create 0x0a010001 
Create 0x0a010001
==> 4: task_create TA1 1
Creating task with:  Preemption: yes Timeslicing: no
Task (TA1) created: id=0x0a010001, priority=1
Task (TA1) starting: id=0x0a010001, priority=1
  CPU 0: Thread Heir: 0x0a010001 priority=1
  CPU 0: Thread Executing: 0x0a010001 priority=1
==> 5: task_suspend TA1
Suspending task (0x0a010001)
  CPU 0: Thread Heir: 0x09010001 priority=255
  CPU 0: Thread Executing: 0x09010001 priority=255
==> 6: task_create TA1 2
Creating task with:  Preemption: yes Timeslicing: no
Task (TA1) created: id=0x0a010002, priority=2
Task (TA1) starting: id=0x0a010002, priority=2
  CPU 0: Thread Heir: 0x0a010002 priority=2
  CPU 0: Thread Executing: 0x0a010002 priority=2
==> 7: task_resume TA1
Resuming task (0x0a010001)
  CPU 0: Thread Heir: 0x0a010001 priority=1
  CPU 0: Thread Executing: 0x0a010001 priority=1
==> 8: echo "*** END OF TEST 04 ***"
*** END OF TEST 04 ***
==> 9: exit
